x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.test.ts

```diff
@@ -69,21 +69,6 @@ const mockDataClientSearchDefault = jest.fn().mockImplementation(
       },
     })
 );
-const mockSearchSourceGetFieldDefault = jest.fn().mockImplementation((key: string) => {
-  switch (key) {
-    case 'fields':
-      return ['date', 'ip', 'message'];
```

Comment:
This mock is not needed, as the Reporting code is no longer calling `getField('fields')` to get the column names.